onnxruntime/core/framework/bfc_arena.cc

```diff
@@ -797,7 +797,7 @@ void BFCArena::DumpMemoryLog(size_t num_bytes) {
   LOGS_DEFAULT(INFO) << "Stats: \n"
                      << stats_.DebugString();
 }
-
+#ifdef ENABLE_STREAM
```

Comment:
It's better to add a prefix to such names to avoid conflicts. You added this macro def to all external libraries, for example protobuf, which is not needed.

Reply:
Yeah, this should be renamed to something like ORT_ENABLE_STREAM

Reply:
Thanks for the suggestion. Let me talk to Cheng about this name change once he is back

Reply:
Renamed